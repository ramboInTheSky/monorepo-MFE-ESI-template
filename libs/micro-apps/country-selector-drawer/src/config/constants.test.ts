import {
    COUNTRY_SELECTOR_DRAWER_BREAKPOINT,
    CHEVRON_ICON_URL,
    COUNTRY_SELECTOR_URL,
    COUNTRY_LANG_SELECTOR_DATA_GA,
    COUNTRY_SELECT_DROPDOWN_DATA_GA,
} from "./constants"

describe("Constants", () => {
    it("should match COUNTRY_SELECTOR_DRAWER_BREAKPOINT", () => {
        expect(COUNTRY_SELECTOR_DRAWER_BREAKPOINT).toMatchSnapshot()
    })

    it("should match CHEVRON_ICON_URL", () => {
        expect(CHEVRON_ICON_URL).toMatchSnapshot()
    })

    it("should match COUNTRY_SELECTOR_URL", () => {
        expect(COUNTRY_SELECTOR_URL).toMatchSnapshot()
    })

    it("should match COUNTRY_LANG_SELECTOR_DATA_GA", () => {
        expect(COUNTRY_LANG_SELECTOR_DATA_GA).toMatchSnapshot()
    })

    it("should match COUNTRY_SELECT_DROPDOWN_DATA_GA", () => {
        expect(COUNTRY_SELECT_DROPDOWN_DATA_GA).toMatchSnapshot()
    })
})
