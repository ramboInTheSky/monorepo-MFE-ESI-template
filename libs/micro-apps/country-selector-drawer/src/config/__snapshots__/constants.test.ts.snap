// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants should match CHEVRON_ICON_URL 1`] = `"/icons/shared/chevron.svg"`;

exports[`Constants should match COUNTRY_LANG_SELECTOR_DATA_GA 1`] = `"Country Language"`;

exports[`Constants should match COUNTRY_SELECT_DROPDOWN_DATA_GA 1`] = `"Country Select"`;

exports[`Constants should match COUNTRY_SELECTOR_DRAWER_BREAKPOINT 1`] = `"524px"`;

exports[`Constants should match COUNTRY_SELECTOR_URL 1`] = `"/countryselect"`;
